.github/workflows/code-engine-cleanup.yml

```diff
@@ -0,0 +1,70 @@
+name: Container Registry and Code Engine clean up
+
+on:
+  workflow_call:
+    inputs:
+      code_engine_project:
+        required: true
+        type: string
+      docker_image_name:
+        required: true
+        type: string
+      ibmcloud_region:
+        default: us-south
+        required: false
+        type: string
+      ibmcloud_resource_group:
+        default: Quantum Community
+        required: false
+        type: string
+      pr_tag:
+        default: pr-${{ github.event.pull_request.number }}
+        required: false
+        type: string
+    secrets:
+      ibmcloud_account:
+        required: true
+      ibmcloud_api_key:
+        required: true
+
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+    env:
+      APP_NAME: ${{ inputs.docker_image_name }}-${{ inputs.pr_tag }}
+      DOCKER_IMAGE_TAG: icr.io/${{ inputs.container_registry_namespace }}/${{ inputs.docker_image_name }}:${{ inputs.pr_tag }}
+    
+    steps:
+      - name: GH deployment status (deactivate)
+        id: gh_deployment
+        uses: bobheadxi/deployments@v0.6.2
+        with:
+          step: deactivate-env
+          token: ${{ github.token }}
+          env: ${{ env.APP_NAME }}
```

Comment:
We should deactivate it?
Because we use the same `env` for all the PRs.

Reply:
It's a good question, in the code that I was preparing for this I had the next line:
https://github.com/Qiskit/gh-actions/blob/clean-up-workflow/.github/workflows/code-engine-clean-up.yml#L44

But I'm not sure about which will be the final result of this.

Reply:
A little update about the change that we needed to merge after checking that this was not working as expected:

https://github.com/Qiskit/gh-actions/pull/11

We can discuss the decision about the naming.